pages/osx/plutil.md

```diff
@@ -0,0 +1,23 @@
+# plutil
+
+> View, convert, validate, or edit property list ("plist") files.
```

Comment:
Note: I found some man pages like https://www.theiphonewiki.com/wiki/Plutil#Apple.27s_plutil but is not synced with the mac options.